runtime/common/src/main/java/org/apache/nemo/runtime/common/metric/MetricUtils.java

```diff
@@ -79,14 +81,28 @@
   private MetricUtils() {
   }
 
+  /**
+   * Method to derive db credentials.
+   * @return credentials.
+   */
+  private static String getCreds() {
+    try {
```

Comment:
I suggest adding comment(s) here to indicate this password is not for real authentication.

Reply:
Thanks! I've reflected your comment. 😄